_posts/2020-06-30-sparklyr-1.3.0-released/sparklyr-1.3.0.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "sparklyr 1.3: Higher-order Functions, Avro, sparklyr.flint, EMR 6.0, Spark 3.0, and much more"
+description: |
+  Sparklyr 1.3 is now available, featuring exciting new functionalities such as integration of Spark higher-order functions and data import/export in Avro and in user-defined serialization formats.
+author:
+  - name: Yitao Li
+    url: https://github.com/yl790
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr-1.3
+date: 06-30-2020
+categories:
+  - R
+  - Packages/Releases
+  - Distributed Computing
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/sparklyr-1.3.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+What would be even better than [sparklyr](https://sparklyr.ai) 1.2? Many brilliant minds from RStudio, Databricks, and the broader open-source community have contemplated this question for months. Now they have an answer -- sparklyr 1.3 is on [CRAN](https://cran.r-project.org/web/packages/sparklyr/index.html) today!
+
+This new version adds the following major features:
+
+ * [Higher-order Functions](#higher-order-functions) to easily manipulate arrays and structs.
+ * Support for Apache [Avro](#avro), a row-oriented data serialization framework.
+ * [Custom Serialization](#custom-serialization) using R functions to read and write any data format.
+
+To install `sparklyr` 1.3 from CRAN, run
```

Comment:
I think we should have a pointer to a "much more" section where we detail the additional features that are being developed, EMR, etc.

```suggestion
 * And other ongoing [Efforts and Improvements](#efforts-and-improvements) like initial support for Flint, EMR 6.0 and Spark 3.0.

To install `sparklyr` 1.3 from CRAN, run
```